snorkel/models/__init__.py

```diff
@@ -36,7 +36,7 @@ class NewType(Context):
     import snorkel.models
 """
 from .meta import SnorkelBase, SnorkelSession, snorkel_engine, snorkel_postgres
-from .context import Context, Document, Sentence, TemporarySpan, Span
+from .context import Context, Document, Webpage, Sentence, Table, Cell, Phrase, TemporarySpan, Span, TemporaryImplicitSpan, ImplicitSpan
```

Comment:
We'd like to keep the core schema as simple as possible. Please move additional models/datatypes to a Fonduer module.